src/quafel/pipelines/visualization/nodes.py

```diff
@@ -111,6 +117,145 @@ def extract_framework_name_from_id(identifier):
         return identifier.replace("fw", "").capitalize().replace("_", " ")
 
 
+def heatmap_viz(x, y, z, z_title, x_title, log_x, y_title, log_y, plot_title):
```

Comment:
It might be helpful to add the data types for the different parameters. Just from reading the code I am currently guessing, that `log_x` and `log_y` might be boolean.

Reply:
resolved